benches/oneshot/ipa.rs

```diff
@@ -3,16 +3,17 @@ use raw_ipa::error::Error;
 use raw_ipa::ff::Fp32BitPrime;
 use raw_ipa::ipa_test_input;
 use raw_ipa::protocol::ipa::ipa;
+// use raw_ipa::protocol::ipa::{ipa_wip_malicious};
+
 use raw_ipa::protocol::{BreakdownKey, MatchKey};
-use raw_ipa::secret_sharing::replicated::semi_honest::AdditiveShare as Replicated;
 use raw_ipa::test_fixture::input::GenericReportTestInput;
 use raw_ipa::test_fixture::{Runner, TestWorld, TestWorldConfig};
 use std::num::NonZeroUsize;
 use std::time::Instant;
 
 #[tokio::main(flavor = "multi_thread", worker_threads = 3)]
 async fn main() -> Result<(), Error> {
-    const BATCHSIZE: usize = 100;
+    const BATCHSIZE: usize = 1000;
```

Comment:
This will change Ben Case's benchmarks =). We should warn him!

Reply:
I will change it back to 100?